### ISSUE TITLE
Example in README doesn't work

### ISSUE THREAD

**Issue by typebrook** [colinmcintosh/SLIDER-cli#7]
@colinmcintosh 
Example like
```
./slider-cli --satellite=goes-16 --sector=conus --product=geocolor -z=2
```
doesn't work, may RAMMB changes some image URLs

**Comment by oscaredel** [comment]
Checkout: https://github.com/colinmcintosh/SLIDER-cli/pull/8

**Comment by colinmcintosh** [comment]
This was fixed by #8. Thanks @oscaredel for the fix! Feel free to let me know if you are still having any issues.
